### PR TITLE
Hcap 256 cypress in pipeline

### PR DESCRIPTION
**Pull request by Zenen-freshworks**
Cypress test suite can be activated by tagging cypress or on PR into master branch!

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -1,11 +1,44 @@
...
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Make env
+        run: |
+          cat > .env << EOF
+          ${{ secrets.ENVFILE }}
+          EOF
+          cat .env
```
Comment (ashtonmeuser) [inline .github/workflows/test.yml:25]: Seems like a red flag to `cat` this file.

**Review by ashtonmeuser — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -1,11 +1,43 @@
...
+
+      - name: Build Locally
+        run: make local-build
+
+      - name: Run locally
+        run: make local-run &
+
+      - name: Waiting for server to spin up
+        run: |
+          while ! nc -z localhost 4000; do
```
Comment (ashtonmeuser) [inline .github/workflows/test.yml:34]: I hope this doesn't put GH Actions in a state where it runs forever and eats into our compute quota. I guess `make local-run` would have to fail for that to be the case which would kill the Action.

Comment (ashtonmeuser) [reply .github/workflows/test.yml:34]: On second thought, Docker Compose doesn't fail if a service fails, so I think this is a possibility.

**Review by ashtonmeuser — APPROVED** [review]

File: .github/workflows/test.yml (hunk 2 above)
Comment (fwkendall) [reply .github/workflows/test.yml:34]: Logging another ticket to look into this :)

**Review by fwkendall — APPROVED** [review]

**Pull request merged**